docs/release-note-1.5.0.md

```diff
@@ -0,0 +1,168 @@
+# dcrstakepool v1.5.0
+
+dcrstakepool release 1.5.0 contains all development work completed since
+[v1.2.0](https://github.com/decred/dcrstakepool/releases/tag/v1.2.0) (September 2019).
+Since then, 7 contributors have produced and merged 37 pull requests.
+Changes include minor GUI enhancements, improved HTTP request logging,
+some streamlining of existing code, and various bug-fixes.
+
+dcrstakepool 1.5.0 requires dcrd/dcrwallet 1.5.0 or later.
+
+## Config Changes
+
+### Removed
+
+All of the following config items were removed by [#546](https://github.com/decred/dcrstakepool/pull/546) as they were deprecated in the previous dcrstakepool release.
+These config items **must** be removed from `dcrstakepool.conf`.
+dcrstakepool will not start if these config items are set.
+
+|Config|Reason|
+|------|------|
+|`wallethosts`, `walletusers`, `walletpasswords` and `walletcerts`|dcrstakepool no longer contacts dcrwallet directly. All comms are now routed through stakepoold.|
+|`enablestakepoold`|stakepoold is always required. dcrstakepool cannot function without it.|
+|`maxvotedage`|The last N voted tickets are now displayed rather than tickets which voted in the last N days.|
+|`minservers`|This value is now hard-coded. Mainnet requires at least two back-end servers, testnet and simnet only need one.|
+|`datadir`|This value was unused. dcrstakepool does not write any data to disk.|
+
+### Deprecated
+
+It is **recommended** to remove these config items from `dcrstakepool.conf` **and** `stakepoold.conf`.
+dcrstakepool and stakepoold will ignore these config items and log a warning if they are set.
+They will be removed completely in the next release.
+
+|Config|PR|Reason|
+|------|--|------|
+|`Profile`, `CPUProfile` and `memprofile`|[#545](https://github.com/decred/dcrstakepool/pull/545)|These config items were not used at all by dcrstakepool|
+
+### Added
+
+No new config items have been added in 1.5.0.
+
+## Recommended Upgrade Path
+
+1. Build dcrstakepool and stakepoold v1.5.0
+1. Build dcrwallet and dcrd v1.5.0
+1. Announce maintenance and shut down dcrstakepool
+1. Perform an upgrade of each back-end server, one at a time
+   1. Stop stakepoold
+   1. Stop dcrwallet
+   1. Stop dcrd
+   1. Install latest dcrd binary and start
+   1. Install latest dcrwallet binary and start
+   1. Make required changes to stakepoold.conf (detailed [above](#config-changes))
+   1. Install latest stakepoold binary and start
```

Comment:
I feel like we should say something about not starting stakepoold until dcrd and dcrwallet have synced with the network.  I think we had a problem here last time everyone updated. Or is there anything we could do in the code? should make an issue?